composer.json

```
@@ -1,19 +1,23 @@
 {
-    "name": "yiisoft/_____",
+    "name": "yiisoft/data-cycle",
     "type": "library",
-    "description": "_____",
+    "description": "Cycle ORM query adapter for yiisoft/data data providers",
```

```suggestion
    "description": "Cycle ORM query adapter for yiisoft/data",
```

Package contain not only data providers, writer also.